hydrolib/core/io/net/models.py

```diff
@@ -737,6 +737,7 @@ def _add_branch(
         name: str = None,
         branch_order: int = -1,
         long_name: str = None,
+        force_midpoint: bool = True,  # argument to control if a midpoint will be forced on the branch
```

Comment:
Please do documentation in a function docstring, not in a comment.

Reply:
Hi @arthurvd, docstrings are added to the function. please see the commit below.